### DISCUSSION TITLE
GLB support

### DISCUSSION THREAD

**Discussion by ZikZakFR** [mastodon/mastodon#23214] [General]
Mastodon supports various image formats, would it be possible to support [GLB](https://www.khronos.org/)?

**Comment by trwnh** [comment]
GLB appears to be a 3d file format which is probably out of scope for Mastodon

**Comment by ZikZakFR** [comment]
You are correct it is a 3D file format.
I'm sure that people using both Mastodon and Blender would appreciate this format (instead of using massive gif or video files).
